ocs_ci/deployment/flexy.py

```diff
@@ -88,8 +88,28 @@ def deploy_prereq(self):
             self.clone_and_unlock_ocs_private_conf()
             config.FLEXY['VARIABLES_LOCATION'] = self.template_file
         config.FLEXY['INSTANCE_NAME_PREFIX'] = self.cluster_name
+        config.FLEXY['LAUNCHER_VARS'].update(
+            self.get_installer_payload()
+        )
         self.merge_flexy_env()
 
+    def get_installer_payload(self, version=None):
+        """
+        A proper installer payload url required for flexy
+        based on DEPLOYMENT['installer_version'].
+        If 'nigtly' is present then we will use registry.svc to get latest
+        nightly else if '-ga' is present then we will look for
+        ENV_DATA['installer_payload_image']
+
+        """
+        payload_img = {"installer_payload_image": None}
+        vers = version or config.DEPLOYMENT['installer_version']
+        installer_version = expose_ocp_version(vers)
+        payload_img['installer_payload_image'] = (
+            ":".join([constants.REGISTRY_SVC, installer_version])
```

Comment:
you could just do str formationg like f"{constants.REGISTRY_SVC}:{installer_version}"

Reply:
Agree, that would been simpler. Since I have approvals m keeping the same. Let me know if you need the change.

Reply:
No need for change ;) just tip for next time ;)